### PR TITLE
Update README. Add dashes.

### PR DESCRIPTION
**Pull request by Gaitorius**
## If this PR were a GIF, what would it be?
Alt Text

## Description of change(s)
Two new rules.  Dashes.

## Diligence checklist (omit non-applicable items):
- [x] Has Readme/Wiki/documentation been updated?
- [ ] Are you a lumberjack?
- [x] Is that OK?
- [x] Is this code production ready?

### CODE REVIEW

**Review by mathisto — APPROVED** [review]
👍

**Comment by fiteclub** [comment]
Double gif!  Very very nice.

File: README.md [hunk 1]
```diff
@@ -11,9 +11,11 @@ Make a PR. Fux wit dis repo. Add things. Delete things. Screw things up so you c
...
-5. ????
-6. PROFIT!!!
+- Thou shalt ask ONLY ridiculous questions, and ridiculous shall thine questions be.
+- All Pull Requests must be accompanied by an animated gif. No exceptions.
+- three sir!
+- Edit [WeAreGo.md](docs/WeAreGo.md) checking the box next to your name to prove your karate & friendship. (**test if you can commit & push**)
+- All roads lead to Rome
+- Get your hands dirty, try for thirty - After that, ask for Matt
+- ????
+- PROFIT!!!
```
Comment (todtb) [inline README.md]: Not a blocker, but did you know GH markdown lets you use numbers like this:

1.
1.
1.
1.

**Review by todtb — APPROVED** [review]
Looks good, check out the GH markdown docs for numbered and bulleted lists

File: README.md (hunk 1 above)
Comment (todtb) [reply README.md]: which is 

```markdown
1.
1.
1.
1.
```

or in ruby
```ruby
def method

end
```

File: README.md [hunk 2]
```diff
@@ -11,9 +11,11 @@ Make a PR. Fux wit dis repo. Add things. Delete things. Screw things up so you c
 
 ## Rules
 Hereth be yon rules:
-1. Thou shalt ask ONLY ridiculous questions, and ridiculous shall thine questions be.
-2. All Pull Requests must be accompanied by an animated gif. No exceptions.
-3. three sir!
```
Comment (dmohar) [inline README.md:16]: three sir should be #5 for the joke to work!

**Pull request merged**